sycl/source/detail/graph_impl.hpp

```diff
@@ -620,6 +639,9 @@ class graph_impl {
   /// Controls whether we skip the cycle checks in makeEdge, set by the presence
   /// of the no_cycle_check property on construction.
   bool MSkipCycleChecks = false;
+
+  /// Force to use an emulated backend
+  bool MEmulationModeForced = false;
```

Comment:
Can this be `const` or a we expected to change it after the object is created?

Reply:
This member is used for debugging/testing purposes and is set explicitly during this lifetime of the object.